core/src/main/java/org/jenkins/ui/icon/IconSet.java

```diff
@@ -116,9 +124,17 @@ public static String getSymbol(String name, String title, String tooltip, String
         symbol = symbol.replaceAll("(<title>)[^&]*(</title>)", "$1$2");
         symbol = symbol.replaceAll("(class=\")[^&]*?(\")", "$1$2");
```

Comment:
Why keep an empty class attibute?
The result is when classes are passed
that we have 
`<svg class="class1 class" class="" ...>`